src/molecules/content/ComparatorSingleModal.tsx

```diff
@@ -0,0 +1,180 @@
+import React, { Component } from 'react';
+import {
+  Dimensions,
+  ImageRequireSource,
+  LayoutChangeEvent,
+  StyleSheet,
+  Text,
+  View,
+} from 'react-native';
+import { layoutStyles } from 'src/utils/styles';
+import { registeredTextStyle } from 'src/utils/textStyles';
+import { Chip } from 'src/molecules/Chip';
+import { ChipLarge, ChipLargeProps } from 'src/molecules/ChipLarge';
+import { colors } from 'src/utils/colors';
+
+const styles = StyleSheet.create({
+  Wrapper: {
+    height: 72,
+    ...layoutStyles.ColumnJustifyStart,
+    alignItems: 'flex-start',
+    width: '60%',
+  },
+  Title: registeredTextStyle('title'),
+  Content: {
+    ...layoutStyles.RowAlignCenter,
+    justifyContent: 'flex-start',
+  },
+  BottomLabel: {
+    ...registeredTextStyle('body'),
```

Comment:
Should I move this to a constant like `const body = registeredTextStyle('body')` or `import { body } from ...`. I'll research this afternoon.

Reply:
no, this is fine. It's only called once, not with every render()